### DISCUSSION TITLE
labels

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-guides#385] [General]
I have labelled the issues as following: 

- `type: enhancement` => New Guides
- `type: improvements` => Fixes, more languages, small corrections to existing guides. 
- `bug` =>  something in the guide does not work .
- `infrastructure` => changes to the tooling we use to publish guides. E.g. feature to add a sitemap.xml etc.
- `review` old review tag. 

We should prioritise bugs. 

@burtbeckwith @ilopmar let me know if you find concerns with such labelling strategy.

**Comment by ilopmar** [comment]
Please don't remove the `review` label because we have an open "contest" with the Community: https://micronaut.io/2021/07/02/community-project-micronaut-guides-review/
